src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+  height: 40px;
+}
+
+.nav {
+  height: 60px;
+  margin-right: 50px;
+  align-items: center;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin: 0 10px;
+  padding: 0;
+  justify-content: center;
+  display: flex;
+  align-items: center;
```

Comment:
please, order your css properties in a [better way](https://9elements.com/css-rule-order/)